tests/test_plotfiledata.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+import os
+from pathlib import Path
+
+
+import amrex.space3d as amr
+
+
+def test_plotfiledata_read():
+    parent_path = Path(os.path.abspath(__file__)).parent
+    plt = amr.PlotFileData(str(parent_path / "projz04000"))
+
+    assert plt.spaceDim() == 3
+    assert plt.time() == 56649980960510.086
+    assert plt.finestLevel() == 0
+    assert plt.refRatio(0) == 0
+    assert plt.coordSys() == amr.CoordSys.cartesian
+
+    probDomain = plt.probDomain(0)
+    probSize = plt.probSize()
+    probLo = plt.probLo()
+    probHi = plt.probHi()
+    cellSize = plt.cellSize(0)
+    varNames = plt.varNames()
+    nComp = plt.nComp()
+    nGrowVect = plt.nGrowVect(0)
+
+    # assert(probDomain == amr.Box([0,0,0],[1024-1,256-1,1-1])) # TypeError: unhashable type: 'instancemethod'.
```

Comment:
@ax3l Is this the correct way to test the contents of a `amrex.space3d.Box`? I get a `TypeError: unhashable type: 'instancemethod'` error when I try this.